### ISSUE TITLE
Add the ability to launch directly with a preset video mode instead of having to choose every time

### ISSUE THREAD

**Issue by Yrouel** [jhhoward/MicroWeb#32]
As per title it would be nice if you could set somewhere in which mode MicroWeb will launch by default thus bypassing the choice menu.

My suggestion is if possible to integrate with MTCP.CFG adding a [MicroWeb] section, if that's not possible either having a MICROWEB.CFG or a MICROWEB environmental variable could be valid solutions.

A command line option could also be a solution of course, but personally I think it would be much nicer to be able to run MicroWeb by simply typing microweb and having it automatically fetch the default option instead of having to type microweb -something videomode which would require an extra wrapper batch file to shorten it back up

**Comment by giobbino** [comment]
> [...] having a MICROWEB.CFG or a MICROWEB environmental variable could be valid solutions.
> A command line option could also be a solution [...]

I agree, the menu should appear only if there's no .cfg file, and microweb could create the .cfg based on the first choice; 
maybe also a command line option to override the .cfg could be useful if you want to run it in a different mode. 
Oh, well, just an idea.

**Comment by steviator** [comment]
I'd prefer the environment variable approach personally, the path to MTCP.CFG and HTTP_PROXY are already specified via variables. 
Support for a VIDEO_MODE variable could probably be added more easily than parsing a configuration file and wouldn't require disk access on painfully slow storage devices.
